cogeo_mosaic/utils.py

```diff
@@ -177,6 +177,7 @@ def create_mosaic(
     dataset_list: Sequence[str],
     minzoom: int = None,
     maxzoom: int = None,
+    quadkey_zoom: int = None,
```

Comment:
Note there is a potential risque when a user will pass `quadkey_zoom` and `version==0.0.1`. But that's only on the API side (not on the CLI) so it's unlikely.

Reply:
will be taken care of in #57